examples/vision/image_classification_from_scratch.py

```diff
@@ -2,7 +2,7 @@
 Title: Image classification from scratch
 Author: [fchollet](https://twitter.com/fchollet)
```

Comment:
Change author to:
"[fchollet](https://twitter.com/fchollet), updated by <names of the updaters>"

Reply:
Done!